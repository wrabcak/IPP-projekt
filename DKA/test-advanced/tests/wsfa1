(
{s,q1,q2,f},
{'a','b'},
{
s'a' -> q1,
s'b'->q2,
q1'a'->q2,
q1'b'->f
},
s,
{f}
)
