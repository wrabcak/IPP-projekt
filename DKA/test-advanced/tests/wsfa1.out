(
{f, q1, qFALSE, s},
{'a', 'b'},
{
f 'a' -> qFALSE,
f 'b' -> qFALSE,
q1 'a' -> qFALSE,
q1 'b' -> f,
qFALSE 'a' -> qFALSE,
qFALSE 'b' -> qFALSE,
s 'a' -> q1,
s 'b' -> qFALSE
},
s,
{f}
)